app/uk/gov/hmrc/lisaapi/controllers/AccountController.scala

```diff
@@ -213,13 +213,41 @@ class AccountController extends LisaController {
     service.closeAccount(lisaManager, accountId, closeLisaAccountRequest).map { result =>
       result match {
         case CloseLisaAccountSuccessResponse(accountId) => {
+
+          auditService.audit(
+            auditType = "accountClosed",
+            path = getCloseEndpointUrl(lisaManager,accountId),
+            auditData = closeLisaAccountRequest.toStringMap + ("lisaManagerReferenceNumber" -> lisaManager)+ ("accountID" -> accountId)
+          )
+
           val data = ApiResponseData(message = "LISA Account Closed", accountId = Some(accountId))
 
           Ok(Json.toJson(ApiResponse(data = Some(data), success = true, status = 200)))
         }
-        case CloseLisaAccountAlreadyClosedResponse => Forbidden(Json.toJson(ErrorAccountAlreadyClosed))
-        case CloseLisaAccountNotFoundResponse => NotFound(Json.toJson(ErrorAccountNotFound))
-        case _ => InternalServerError(Json.toJson(ErrorInternalServerError))
+        case CloseLisaAccountAlreadyClosedResponse => {
+          auditService.audit(
+            auditType = "accountNotClosed",
+            path = getCloseEndpointUrl(lisaManager,accountId),
+            auditData = closeLisaAccountRequest.toStringMap + ("lisaManagerReferenceNumber" -> lisaManager)+ ("accountID" -> accountId) + ("reasonNotClosed" -> ErrorAccountAlreadyClosed.errorCode)
```

Comment:
You could simplify these lines by adding a single map containing all of the keys (see life event controller for example)